static/js/src/stripe/set-modal-info.js

```diff
@@ -0,0 +1,118 @@
+function getCardImgFilename(brand) {
+  switch (brand) {
+    case "visa":
+      return "832cf121-visa.png";
+    case "mastercard":
+      return "83a09dbe-mastercard.png";
+    case "amex":
+      return "91e62c4f-amex.png";
+    default:
+      return false;
+  }
+}
+
+function getProductName(slug) {
```

Comment:
Wouldn't this be better as a mapping object?

``` javascript
projectNames = {
  uai-essential-desktop: "UA Infra Essential Desktop"
}

...

if (slug in projectNames) {
  projectNames[slug];
}
```

Reply:
Oh nice - yeah that would work